dm_api_account/apis/account_api.py

```diff
@@ -69,38 +62,30 @@ def get_v1_account(
             path='/v1/account',
             **kwargs
         )
+        if validate_response and response.status_code == 200:
```

Comment:
Надо убрать на след неделе будет другим образом чекать статус коды
```suggestion
        if validate_response:
```